README.md

```diff
@@ -173,15 +173,32 @@ The service architecture is currently defined [on confluence].
 
 ## Putting the application into maintenance mode
 
-If we need to take the service offline for any reason (for example, we're
-investigating an issue, or we want to deploy a journey-breaking change), it's
-possible to put ther service into maintenance mode.
+There are two potential reasons why we might want to turn development mode on.
+Either prior to a deploy of a journey breaking change, or until further notice
+(the service is closed for some reason).
 
-To do this, we set the environment variable `MAINTENANCE_MODE` to a value of
-`1`. If you know when the service will be operational again, you can also set a
-`MAINTENANCE_MODE_AVAILABILITY_MESSAGE` environment variable with a
+### Turning development more on temporarily
+
+To do this, run the following command:
+
+```bash
+bin/set-maintenance-mode ENVIRONMENT "OPTIONAL_AVAILABILITY_MESSAGE"
+```
+
+Where `ENVIRONMENT` is the environment you're turning maintenance mode on for,
+and `OPTIONAL_AVAILABILITY_MESSAGE` is a human-readable value of when to expect
+the service to be operational again as a full sentence, for example
+`The service will be available from 2pm today.`
+
+Development mode will then be automatically disabled on the next deploy.
+
+### Turning development mode on until further notice
+
+This involves us settinh the environment variable `MAINTENANCE_MODE` to a value
+of `1`. If you know when the service will be operational again, you can also set
+a `MAINTENANCE_MODE_AVAILABILITY_MESSAGE` environment variable with a
 human-readable value of when to expect the service to be operational again as a
-full sentence, for example `The service will be available from 2pm today.` or
+full sentence, for example
 `Private beta has ended. You will be able to use the service again in November`.
```

Comment:
Do you think it makes sense to explain why we don't want to use the script in this case?

Reply:
```suggestion
`Private beta has ended. You will be able to use the service again in November`.

We do this as a manual step to ensure the settings are preserved over deployments.
```

Reply:
Did you see this @pezholio?

Reply:
Whoops, no. I did wonder why you'd said "a couple of things", but I could only see one 😆